src/sqlfluff/core/rules/base.py

```diff
@@ -221,6 +228,34 @@ class RuleContext:
     path: Optional[pathlib.Path]
     templated_file: Optional[TemplatedFile]
 
+    @cached_property
+    def surrogates(self):
+        """Returns a Surrogates object that simplifies writing rules."""
+        return Surrogates(self)
+
+
+class Surrogates:
+    """Provides access to surrogate objects that simplify writing rules."""
+
+    def __init__(self, context: RuleContext):
+        self.context = context
+
+    @cached_property
+    def segment(self) -> "Segments":
+        """Returns a Segments object for context.segment."""
+        # Import here to avoid circular dependency.
+        from sqlfluff.core.rules.surrogates import Segments
+
+        return Segments(self.context.templated_file, self.context.segment)
+
+    @cached_property
+    def raw_stack(self) -> "Segments":
```

Comment:
worth having a parent_stack method as well

Reply:
👍  Easy to add.

Reply:
Adding now.